src/artemis/ispyb/store_in_ispyb.py

```diff
@@ -102,6 +100,8 @@ def update_grid_scan_with_end_time_and_status(
             params["parentid"] = datacollection_group_id
             params["endtime"] = end_time
             params["run_status"] = run_status
+            if reason is not None and reason != "":
+                params["comments"] += " " + run_status + " reason: " + reason
```

Comment:
Nit:
```suggestion
                params["comments"] += f" {run_status} reason: {reason}"
```
Feels a bit cleaner to me